### PR TITLE
Feat/contacts

### PR DESCRIPTION
**Pull request by askbk**
This adds API endpoints for fetching contacts and fetching and declining contact requests.

### CODE REVIEW

File: src/components/user/userRoutes.js [hunk 1]
```diff
@@ -1,20 +1,45 @@
...
     router.post('/:id/contact', (rq, rs, n) => {
         users.contact(rq, rs, n);
     });
 
+    // Decline a contact request from the user
+    router.post('/:id/contact/decline', (rq, rs, n) => {
+        users.declineRequest(rq, rs, n);
+    });
+
+    // Send a contact request to the user
```
Comment (edvardvb) [inline src/components/user/userRoutes.js:32]: Duplicate endpoint?

File: src/components/user/userRoutes.js [hunk 2]
```diff
@@ -1,20 +1,45 @@
...
+    });
+
+    // Get contact requests (both sent and received)
+    router.get('/:id/contacts/requests', (rq, rs, n) => {
+        users.getContactRequests(rq, rs, n);
+    });
+
+    // Get contacts
+    router.get('/:id/contacts', (rq, rs, n) => {
+        users.getContact(rq, rs, n);
```
Comment (edvardvb) [inline src/components/user/userRoutes.js]: `getContact_s_`

File: src/components/user/userAPI.js [hunk 3]
```diff
@@ -243,6 +244,171 @@ class UserAPI {
...
+
+            res.status(200).send({
+                success: true,
+                message: 'Contact request accepted.'
+            });
+
+            return true;
+        } catch (e) {
+            res.status(500).send({
+                succuss: false,
```
Comment (98mux) [inline src/components/user/userAPI.js:327]: Misspell

Comment (askbk) [reply src/components/user/userAPI.js:327]: oi haha

**Review by Marcusntnu — APPROVED** [review]

**Review by 98mux — APPROVED** [review]

**Pull request merged**